README.rst

```diff
@@ -52,6 +52,12 @@ interpreter (and only the Python3 package installs scripts)::
 News
 ----
 
+ * 2016-12-21: **Version 0.8.6**. See the `download page
+   <https://pypi.python.org/pypi/stdeb/0.8.6>`__. Bugfixes: fix for
+   applying Python 3 patch files, some tests. Improvements: Added
+   `--debian-version`, `--sign-results`, and `--extend-diff-ignore`
+   CLI flags.
```

Comment:
Please revert version-bumping related changes since this change on its own won't result on a new version and it is unclear atm into which version the change might be released.